test/es-module/test-esm-undefined-cjs-global-like-variables.js

```diff
@@ -38,5 +38,5 @@ assert.rejects(
 
 assert.rejects(
   import('data:text/javascript,require;//.js'),
-  /^(?!use the '\.cjs' file extension).*$/
+  /^(?:(?!use the '\.cjs' file extension)[\s\S])*$/
```

Comment:
Why this change?

Reply:
When I looked here, I got an error for an ```unspelled period character in a regular expression```, so I updated it